batcher/aligned-sdk/src/core/constants.rs

```diff
@@ -1,4 +1,6 @@
 /// Batcher ///
+pub const BATCH_INCLUSION_DELAY: u64 = 36_000; // 3 blocks
+pub const TRANSACTIONS_INCLUSION_DELAY: u64 = 14_000; // 1 block + 2 secs
```

Comment:
These values are not delays, these are timeouts
Also, the name is not clear. 
I'd suggest
`TRANSACTION_WAIT_TIMEOUT`
Also, I don't think it is necessary to use two variables

Reply:
Changed! Left only `TRANSACTION_WAIT_TIMEOUT` with a time of 3 blocks.